etc/config/cobol.defaults.properties

```diff
@@ -0,0 +1,14 @@
+# Default settings for COBOL
+compilers=cobol
+compilerType=cobol
+
+versionFlag=--version
+
+objdumper=objdump
+supportsBinary=true
+supportsBinaryObject=true
+binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|\.plt.*|_dl_relocate_static_pie)$
+supportsLibraryCodeFilter=true
+
+compiler.cobol.name=GNU COBOL
+compiler.cobol.exe=/usr/local/bin/cobc
```

Comment:
Please go with `GnuCOBOL` -because that's what the software is called.
Just in case you somewhere need a link I'd suggest https://www.gnu.org/software/gnucobol/